packages/platforms/inventec/x86-64/x86-64-inventec-d7032q28b/onlp/builds/src/module/src/platform_lib.c

```diff
@@ -0,0 +1,200 @@
+/************************************************************
```

Comment:
I understand that this was copied from an existing platform implementation. 
Please do not propagate these custom interfaces but remove them and use the existing common onlp_file_* APIs.